src/components/ComponentConfiguration.stories.tsx

```diff
@@ -1204,6 +1206,21 @@ export const SelectBoxes: Story = {
       // check that the option labels are in the translations table
       expect(await editForm.findByText('Option label 1')).toBeVisible();
       expect(await editForm.findByText('Second option')).toBeVisible();
+
+      // Check that all translations can be filled
+      const inputs = editForm.getAllByRole('textbox');
+      for (let input of inputs) {
+        await userEvent.type(input, 'manualTranslation');
+        await expect(input).toHaveValue('manualTranslation');
```

Comment:
`expect` should not necessarily be `await`-ed. It shouldn't break anything, but it is a bit confusing. Typically it should be something like `expect(await canvas.findByRole(...)).toBeVisible()` to account for some rendering delays etc.

these existing tests are littered with bad examples, so I can hardly blame you for this.